flytekit/core/map_task.py

```diff
@@ -76,7 +76,11 @@ def __init__(
 
         collection_interface = transform_interface_to_list_interface(actual_task.python_interface, self._bound_inputs)
         self._run_task: PythonFunctionTask = actual_task
-        _, mod, f, _ = tracker.extract_task_module(actual_task.task_function)
+        if issubclass(type(actual_task), PythonInstanceTask):
+            mod = actual_task.task_type
```

Comment:
should this be `actual_task.instantiated_in`?  what happens if there are two of the same notebook tasks, named the same, with the same interface, but in two different .py files?  will there be confusion?

Reply:
used lhs instead